### PR TITLE
Add IPFS Backup

### PR DESCRIPTION
**Pull request by wilsonplau**
SSIA

### CODE REVIEW

File: src/controllers/file.ts [hunk 1]
```diff
@@ -30,6 +32,7 @@ export default class UploadController {
     // Add the file to IPFS
     const { cid } = await ipfs.add(fileContent)
     await ipfs.pin.add(cid)
+    await backupCID(cid.toString())
```
Comment (T-Damer) [inline src/controllers/file.ts:35]: Should we `await` on it? Maybe let's `void` or `timeout` so we don't delay file return?

File: src/helpers/backupCID.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      },
+      {
+        headers: {
+          Authorization: `Bearer ${env.FILEBASE_API_TOKEN}`,
+          'Content-Type': 'application/json',
+        },
+      }
+    )
+  } catch (e) {
+    if (e instanceof AxiosError) console.log(e.message)
```
Comment (T-Damer) [inline src/helpers/backupCID.ts:26]: Let's `console.error` all errors, we should probably use sentry with tags here. Backups are too important, we shouldn't miss anything

File: src/helpers/backupCID.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+export default async function backupCID(cid: string) {
+  try {
+    await axios.post(
+      IPFS_PINNING_URL,
+      {
+        cid,
+        name: cid,
+        origins: [
+          `/ip4/${env.IPFS_IP_ADDRESS}/tcp/4001/p2p/${env.IPFS_ID}`,
```
Comment (T-Damer) [inline src/helpers/backupCID.ts:14]: Don't forget to add this into `1pass`

**Review by T-Damer — APPROVED** [review]
Let's fix a few things and merge

File: src/helpers/backupCID.ts (hunk 3 above)
Comment (wilsonplau) [reply src/helpers/backupCID.ts:14]: Yep, it's already added.

File: src/controllers/file.ts (hunk 1 above)
Comment (wilsonplau) [reply src/controllers/file.ts]: Sure, resolved in [d7f487d](https://github.com/BigWhaleLabs/obss-ipfs-uploader/pull/16/commits/d7f487d96020f9ae7729dcff73ebccc3dd16ebb8).

File: src/helpers/backupCID.ts (hunk 2 above)
Comment (wilsonplau) [reply src/helpers/backupCID.ts]: I'll setup Sentry in a separate PR, but I switched to `console.error` in [72d6806](https://github.com/BigWhaleLabs/obss-ipfs-uploader/pull/16/commits/72d6806c588ca23a6f70f148429f09ad7f59790f).

**Pull request merged**